### DISCUSSION TITLE
Include token account address into NFT model

### DISCUSSION THREAD

**Discussion by fedorerd** [metaplex-foundation/js#304] [Q&A]
Include address of the token account, which currently holds NFT, into NFT / metadata models.

Currently, I am retrieving token account which holds NFT with web3.js methods such as `connection.getTokenAccountsByOwner` or `connection.getTokenLargestAccounts`, passing `mintAddress` which I retreive from NFT / metadata models.

It would be great if I could retreive token account address straight from NFT model.

**Comment by lorisleiva** [comment]
Hi there, you can do this by either passing the `tokenAddress` or `tokenOwner` on the `metaplex.nfts().findByMint()` operation ([See API References](https://metaplex-foundation.github.io/js/types/js.FindNftByMintInput.html)).

**Reply by fedorerd** [reply]
Hey, thanks for your help! Have not mentioned that.
